lib/storefront/klarna.svg

```diff
@@ -0,0 +1,41 @@
+<svg version="1.1" id="klarna" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="82.2px" height="22px" viewBox="0 0 82.2 22" style="enable-background:new 0 0 82.2 22;" xml:space="preserve">
+<title>Klarna Logo</title>
+<desc>Klarna Logo</desc>
+<g>
+	<path style="fill:#CBCBCB;" d="M3.9,0C3,0,2.2,0,1.4,0C0.5,0,0,0.5,0,1.4v9.7c0,3.3,0,6.6,0,9.8c0,0.6,0.1,0.7,0.7,0.7
```

Comment:
if we're adding this to our spritesheet, we don't prepopulate it with a fill; we should be doing this through css